samples/tutorial/Tutorial2.md

```diff
@@ -419,7 +419,7 @@ object RootWorkflow : StatefulWorkflow<Unit, State, Nothing, Any>() {
       // When the state is Todo, defer to the TodoListWorkflow.
       is Todo -> {
         val todoScreen
-        = context.renderChild(TodoListWorkflow, Unit)) {
+        = context.renderChild(TodoListWorkflow, Unit) {
```

Comment:
Lint check failed due to trailing spaces. And while you're here, would you mind moving joining lines 421 and 422?

```kotlin
  val todoScreen = context.renderChild(TodoListWorkflow, Unit) {

Reply:
Totally.